fairlearn/datasets/_fetch_acs_income.py

```diff
@@ -47,7 +47,7 @@ def fetch_acs_income(
         By default, all fairlearn data is stored in '~/.fairlearn-data'
         subfolders.
 
-    as_frame : bool, default=False
+    as_frame : bool, default=True
```

Comment:
Can you add the versionchanged tag with a brief note on the default changing? Same in the other files.

Reply:
Yep! Took the "no deprecation cycle" a bit too literally I guess. 😄